db/seeds/02_zipcode_rates.rb

```diff
@@ -0,0 +1,59 @@
+module ZipcodeRateImporter
+  extend self
+  ZIP_MHA_PATH = File.join(Rails.root, 'db', 'seeds', 'zipcode_rate', 'zip_mha.txt')
+  MHA_RATE_PATH = File.join(Rails.root, 'db', 'seeds', 'zipcode_rate', 'mha_rate.txt')
+  ZIP_CITY_PATH = File.join(Rails.root, 'db', 'seeds', 'zipcode_rate', 'zip_city.csv')
+
+  def zip_mha
+    options = {
+      col_sep: '|',
+      headers_in_file: false,
+      user_provided_headers: %w[zip_code mha_code year date],
+      convert_values_to_numeric: false
+    }
+    SmarterCSV.process(ZIP_MHA_PATH, options)
+  end
+
+  def mha_rate
+    options = {
+      col_sep: '|',
+      headers_in_file: false,
+      user_provided_headers: %w[mha_code date mha_rate mha_rate_grandfathered],
+      convert_values_to_numeric: false
+    }
+    SmarterCSV.process(MHA_RATE_PATH, options).group_by { |record| record[:mha_code] }
+  end
+
+  def zip_city
+    options = {
+      col_sep: ',',
+      headers_in_file: true,
+      user_provided_headers: %w[zip_code city state],
+      convert_values_to_numeric: false
+    }
+    SmarterCSV.process(ZIP_CITY_PATH, options).group_by { |record| record[:zip_code] }
+  end
+
+  def insert_data
+    records = []
+    mha_rates = mha_rate
+    zip_cities = zip_city
+    zip_mha.each do |zip_mha_record|
+      mha_rate_data = mha_rates[zip_mha_record[:mha_code]].first.slice(*%i[mha_rate mha_rate_grandfathered])
```

Comment:
I didn't catch any in this `mhv_rate.csv`, but maybe a TODO comment here about throwing an exception if we have multiple rates associated with a code?

Reply:
how about activerecord validation for that?

Reply:
I think this is a temporary solution until we can get a single consolidated file (sometime in the future I suppose based on comments made in the issue).

It isn't really possible to have more than one MHA for a given zip code. The only reason I'm calling `.first` on it is because the group_by function always returns an array, even when there is only one member in the grouping.